internal/gatewayapi/testdata/backendtrafficpolicy-status-conditions.out.yaml

```diff
@@ -575,11 +574,10 @@ xdsIR:
         mergeSlashes: true
       port: 10080
       routes:
-      - backendWeights:
-          invalid: 1
-          valid: 0
-        directResponse:
```

Comment:
why is `directResponse` getting removed

Reply:
fixed, see https://github.com/envoyproxy/gateway/pull/3212/commits/115fdb1f3692ff97846f523c4706fba25eb59ab8